packages/react/src/Card/Card.tsx

```diff
@@ -106,30 +107,32 @@ const CardRoot = forwardRef<HTMLAnchorElement, CardProps>(
     })
 
     const hasIcon = React.Children.toArray(children).some(
-      child => React.isValidElement(child) && typeof child.type !== 'string' && child.type === CardIcon,
+      child => React.isValidElement(child) && typeof child.type !== 'string' && child.type === CardIcon
     )
 
     return (
-      <a
-        href={href}
-        className={clsx(styles.Card, hasIcon && styles['Card--has-icon'], className)}
-        onMouseEnter={handleMouseEnter}
-        onMouseLeave={handleMouseLeave}
-        onFocus={handleOnFocus}
-        onBlur={handleOnBlur}
-        ref={cardRef}
-        {...props}
-      >
-        {filteredChildren}
+      <div className={clsx(styles.Card, hasIcon && styles['Card--has-icon'], className)} ref={cardRef} {...props}>
+        {React.Children.map(filteredChildren, child => {
+          if (React.isValidElement(child) && typeof child.type !== 'string' && child.type === CardHeading) {
+            return React.cloneElement<CardHeadingProps>(child as React.ReactElement<CardHeadingProps>, {
```

Comment:
Are we ok with this use of cloneElement? Or do you think it would be better to use React Context?

Reply:
Cloning is fine 👍, we tend to use it for these kind of use-cases. Only nit would be to hoist it out of the render and wrap in a callback.